test/e2e/federated-ingress.go

```diff
@@ -182,13 +195,63 @@ var _ = framework.KubeDescribe("Federated ingresses [Feature:Federation]", func(
 	})
 })
 
+// Deletes all Ingresses in the given namespace name.
+func deleteAllIngressesOrFail(clientset *fedclientset.Clientset, nsName string) {
+	IngressList, err := clientset.Extensions().Ingresses(nsName).List(v1.ListOptions{})
+	Expect(err).NotTo(HaveOccurred())
+	orphanDependents := false
+	for _, Ingress := range IngressList.Items {
```

Comment:
found there was an existing function here doing the same thing. Changed the code to reuse that